diff --git a/vcbuild.bat b/vcbuild.bat
index 91f45b7..c3337cc 100644
--- a/vcbuild.bat
+++ b/vcbuild.bat
@@ -114,8 +114,8 @@ echo manually install gyp into %~dp0build\gyp.
 exit /b 1
 
 :have_gyp
-if not defined PYTHON set PYTHON=python
-"%PYTHON%" gyp_uv.py -Dtarget_arch=%target_arch% -Duv_library=%library%
+if not defined PYTHON27 set PYTHON27=C:\Python27\python
+"%PYTHON27%" gyp_uv.py -Dtarget_arch=%target_arch% -Duv_library=%library%
 if errorlevel 1 goto create-msvs-files-failed
 if not exist uv.sln goto create-msvs-files-failed
 echo Project files generated.
